workflow-report/action.yml

```diff
@@ -0,0 +1,75 @@
+name: 'Report failed job to VK Team chat'
+description: 'This action composes a message about the failed job and sends it to VK Teams chat'
+
+inputs:
+  api-url:
+    description: 'Bot API URL'
+    required: true
+  bot-token:
+    description: 'Bot token'
+    required: true
+  chat-id:
+    description: 'Notification chat ID (or stamp from chat URL)'
+    required: true
+  job-steps:
+    description: 'Info about steps that have been run in the job'
+    required: false
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Compose message about job failure
+      id: compose-message
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const steps = ${{ inputs.job-steps }}
+          const event = "${{ github.event_name }}"
+          const headRef = "${{ github.head_ref }}"
+          const ref = "${{ github.ref }}"
+          let branch = ""
+          let failedSteps = ""
+          let failedStepsMsg = `---------------- Failed steps info ------------------- 
+          `
+          let base_url = "https://github.com"
+          
+          if (event == "pull_request") {
+            branch = headRef
+          } else {
+            branch = ref.split("/")[2]
+          }
+          
+          commitString = "${{ github.event.head_commit.message }}".split("\n")[0]
+          
+          Object.keys(steps).forEach(function(key) {
+            if (steps[key]["conclusion"] != "success") {
```

Comment:
I can suppose that steps may contain skipped steps.

Reply:
So we want to report about them? Or we don't?